### ISSUE TITLE
Question: How do I resume a "pending" task?

### ISSUE THREAD

**Issue by alusev** [triniwiz/nativescript-downloader#36]
## Which platform(s) does your issue occur on?
- Android emulator 

I save created task id in a database, and when I restart my app I would like to resume downloading. 
I call `downloader.start` but I never receive callbacks.

**Comment by 7ammer** [comment]
Hi @alusev, 
Did you ever resolve this issue? I posted this a while ago but I'm still non the wiser: https://github.com/triniwiz/nativescript-downloader/issues/37

Cheers :)

**Comment by alusev** [comment]
No, I had to forbid this kind of things. Let me know if you find a solution

**Comment by 7ammer** [comment]
Will do! 👍
